src/main/java/creativei/dao/ReminderDao.java

```diff
@@ -2,9 +2,15 @@
 
 import creativei.entity.Reminder;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 import javax.transaction.Transactional;
+import java.util.Date;
+import java.util.List;
 
 @Transactional
 public interface ReminderDao extends JpaRepository<Reminder,Long> {
+    @Query("from Reminder rem where rem.reminderTime between :to and :from")
```

Comment:
What if any of the date value is null? If todate is null, send all from current date to all